js/index.js

```diff
@@ -72,14 +86,6 @@ $(function() {
     }
   });
 
-  $('a').click(function() {
-    $('li.current').removeClass('current');
-  });
-
-  $('li a').click(function() {
-    $(this).parent().addClass('current');
-  });
-
```

Comment:
You forgot to add this part into your new code. So now highlighting of the selected key doesn't work anymore.